jsre/natto.go

```diff
@@ -0,0 +1,170 @@
+/*
+
+This file is a modified version of https://github.com/robertkrimen/natto/blob/master/natto.go
+
+Package natto is an example/offshoot of otto that implements an event loop (supporting setTimeout/setInterval).
+
+http://godoc.org/github.com/robertkrimen/natto
+
+otto: http://github.com/robertkrimen/otto (http://godoc.org/github.com/robertkrimen/otto)
+
+// The VM has the following functions available:
+//
+//      <timer> = setTimeout(<function>, <delay>, [<arguments...>])
+//      <timer> = setInterval(<function>, <delay>, [<arguments...>])
+//      clearTimeout(<timer>)
+//      clearInterval(<timer>)
+//
+
+*/
```

Comment:
When writing "please remove this comment", I meant "please remove the package comment".